webapp/src/components/modals/create_issue/create_issue.jsx

```diff
@@ -16,7 +16,7 @@ import Input from 'components/input';
 const initialState = {
     submitting: false,
     error: null,
-    repoValue: '',
+    repo: {},
```

Comment:
Are there any risks to initializing this as `null`?

```suggestion
    repo: null,
```

Reply:
@mickmister Updated the initial repo state as `null`: https://github.com/mattermost/mattermost-plugin-github/pull/343/commits/bfb58bcb4ad6db5c1e170f524dc4c996d9f7c110